posts/models.py

```diff
@@ -1,3 +1,42 @@
 from django.db import models
+from django.contrib.auth.models import User
+from base.models import Club
+
 
 # Create your models here.
+
+class Post(models.Model):
+    post_creator = models.ForeignKey(User, on_delete=models.CASCADE)
+    club = models.ForeignKey(Club, on_delete=models.CASCADE)
+    is_approved = models.BooleanField()
+    description = models.TextField(max_length=150)
+    video = models.URLField()
+
+
+class Image(models.Model):
+    post = models.ForeignKey(Post, on_delete=models.CASCADE)
+    image_url = models.URLField()
```

Comment:
1. directly use image=ImageField()
2. The is_public option is in post. not in image

Reply:
yes